target/linux/pistachio/image/Makefile

```diff
@@ -24,11 +24,12 @@ define Device/Default
 endef
 
 define Device/marduk
+  DEVICE_VENDOR := Creator
+  DEVICE_MODEL := Ci40
```

Comment:
Looks right, but why is the block called "marduk"?

Reply:
'Creator Ci40 (marduk)'  0d271cef8071d0698246b4b50a610b5ff718af01

Reply:
```
DEVICE_VENDOR:=Imagination Technologies
DEVICE_MODEL:=Creator Ci40 (VL-62899)
```